src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useThorStreamingProgress.tsx

```diff
@@ -92,20 +92,11 @@ export const useThorStreamingProgress = (
 
         // We don't have a quantity in the streaming swap data, and we never have.
         if (!streamingSwapDataRef.current?.quantity && !updatedStreamingSwapData.quantity) {
-          // This is a special case where it _is_ a streaming swap, but it's optimized over a single block, and so doesn't actually stream.
-          // In this case we never get initial streaming metadata, only the default empty response because it's immediately complete.
-          // An empty response means the streaming swap is complete.
-          if (updatedStreamingSwapData.tx_id) {
-            dispatch(
-              tradeQuoteSlice.actions.setStreamingSwapMeta({
-                hopIndex,
-                streamingSwapMetadata: getStreamingSwapMetadata({
-                  tx_id: updatedStreamingSwapData.tx_id,
-                }),
-                id: confirmedTradeId,
-              }),
-            )
-          }
+          // This is a special case where it _is_ a streaming swap, but it's one of the following cases:
+          // - optimized over a single block, and so doesn't actually stream.
+          // - the streaming swap metadata isn't ready yet
+          //
+          // In both cases its simpler to exit and not render the progress bar.
```

Comment:
Noticing this guy https://viewblock.io/thorchain/tx/cb93705ff185966b79980dc746adee3968fc597c99048d673be1f5afa63de13e still rendered an (indeterminate) progress bar: https://jam.dev/c/688463cf-5814-400e-99fd-3db91d653765

Is this expected?

Reply:
Confirmed to be expected!

Reply:
nice, ticket added for follow-up
https://github.com/shapeshift/web/issues/7851